Frameworks/CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -53,26 +55,12 @@
 #include <Foundation/NSTimeZone.h>
 #include <Foundation/NSDate.h>
 
-struct tzhead {
```

Comment:
Should we document removals with WINOBC comments as well? Or use `!defined(WINOBJC)` or similar?

Is that a silly question because of the scope of the changes in this file?